designer/server/src/lib/forms.js

```diff
@@ -56,6 +63,34 @@ export async function remove(id) {
   return Promise.resolve(true)
 }
 
+/**
+ * Get draft form definition
+ * @param {string} id
+ * @returns {Promise<import('@defra/forms-model').FormDefinition | null>}
```

Comment:
Can drop the `@returns` here if the typed `getJson()` response is working

```suggestion
```

Reply:
Will need to copy the ESLint tweak in https://github.com/DEFRA/forms-manager/pull/61/commits/97bc49cdfad8997181c67ce5492fe4b860417dbb#diff-e1ce717e1179a0db14e90ec5374768a206651ca807db58352991eb7895ed7c9c